applications/nublado/values.yaml

```diff
@@ -675,6 +675,60 @@ secrets:
   # user's lab.
   templateSecrets: true
 
+# If we're installing tutorials at a given site, this cronjob controls
+# their refresh.
+cronjob:
+  # -- Enable tutorials refresh?
+  enabled: false
+
+  # -- Source for repository to clone
+  gitSource: "https://github.com/lsst/tutorial-notebooks"
+
+  # -- Target where repository should land
+  gitTarget: "/project/cst_repos/tutorial-notebooks"
```

Comment:
If we're always cloning at the target mount path perhaps we can omit the` /project/ ` here and construct the env variable using the t`argetVolume.mountPath`? Unless we expect to ever use a different directory?